docs/pages/access-controls/reference.mdx

```diff
@@ -334,10 +334,19 @@ who created the session.
 
 <Details title="Version Warning: Before 8.1" opened={false}>
 Teleport versions prior to 8.1 don't support the roles exemplified below.
```

Comment:
```suggestion
Teleport versions prior to 8.1 don't support the roles shown below.
```
I think "exemplified" is somewhat distracting.

Reply:
Agree.